app/src/main/kotlin/io/sweers/catchup/ui/activity/ImageViewerActivity.kt

```diff
@@ -268,6 +260,14 @@ class ImageViewerActivity : AppCompatActivity() {
   }
 }
 
+private fun Context.dip(units: Int): Int {
+  return TypedValue.applyDimension(
+      COMPLEX_UNIT_DIP,
+      units.toFloat(),
+      resources.displayMetrics
+  ).toInt()
+}
```

Comment:
Do you prefer keeping the dimension in Xml?

Reply:
Nope I'm fine with this!